### PR TITLE
Update to Chromium 97.0.4692.71

### LINKED ISSUE #1788: Updating to Chromium 97.0.4692.71
## Notes for the developer

Once you claim it, it is advisable that you create a [Draft Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) for increased visibility:

GitHub Interface for creating Draft Pull Requests

Finally, make sure to reference this issue in your PR.

Feel free to ask questions and discuss issues here along the way. Thanks!

## Notes for others

Feel free to raise issues or questions throughout the process. However, please refrain from asking ETAs unless no visible progress has been made here or on the developer's PR for a while (e.g. 2 weeks).

### PR DESCRIPTION
**Pull request by Ahrotahn**
Updates to Chromium 97.0.4692.71.  

Notable changes:
* `add-flag-to-stack-tabs.patch` has been removed due to the changes in [[3206542]](https://chromium-review.googlesource.com/c/chromium/src/+/3206542), [[3207236]](https://chromium-review.googlesource.com/c/chromium/src/+/3207236), [[3223435]](https://chromium-review.googlesource.com/c/chromium/src/+/3223435), [[3227713]](https://chromium-review.googlesource.com/c/chromium/src/+/3227713), and [[3230235]](https://chromium-review.googlesource.com/c/chromium/src/+/3230235) which also affected the implementation of `add-flag-to-hide-tab-close-buttons.patch`.
* A new patch to disable the remote optimization guide has been added.  This is to prevent a connection attempt likely caused by the changes in [[3311558]](https://chromium-review.googlesource.com/c/chromium/src/+/3311558)/[[3322592]](https://chromium-review.googlesource.com/c/chromium/src/+/3322592).
* Removal of a privacy review script that would cause a crash due to building without safebrowsing [[3237250]](https://chromium-review.googlesource.com/c/chromium/src/+/3237250).
* client_side_model_loader has finally been removed [[3219837]](https://chromium-review.googlesource.com/c/chromium/src/+/3219837).
* The extension blocklist removal has been updated for the code changes in [[3146978]](https://chromium-review.googlesource.com/c/chromium/src/+/3146978).
* Small change in the search suggestion patch due to [[3212216]](https://chromium-review.googlesource.com/c/chromium/src/+/3212216).

Fixes #1788 

[Blog post](https://chromereleases.googleblog.com/2022/01/stable-channel-update-for-desktop.html)
[Change log](https://chromium.googlesource.com/chromium/src/+log/96.0.4664.110..97.0.4692.71?pretty=fuller&n=10000)

### CODE REVIEW

**Comment by kramred** [comment]
The Beta channel has been updated to [97.0.4692.71](https://chromereleases.googleblog.com/2021/12/beta-channel-update-for-desktop_30.html) for Windows, Mac and Linux.

**Comment by PF4Public** [comment]
Builds and runs on Gentoo

**Comment by Ahrotahn** [comment]
There's an [upstream bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1279532) on linux that causes tabs not to rejoin windows when dragged.  Reverting the changes in [[3243293]](https://chromium-review.googlesource.com/c/chromium/src/+/3243293) and [[3243244]](https://chromium-review.googlesource.com/c/chromium/src/+/3243244) solves the issue for me.

File: patches/extra/ungoogled-chromium/add-flag-to-hide-tab-close-buttons.patch [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
- }
+--- a/chrome/browser/ui/views/tabs/tab.cc
++++ b/chrome/browser/ui/views/tabs/tab.cc
+@@ -972,6 +972,8 @@ void Tab::UpdateIconVisibility() {
+   if (IsActive()) {
+     // Close button is shown on active tabs regardless of the size.
+     showing_close_button_ = true;
++    if (base::CommandLine::ForCurrentProcess()->HasSwitch("hide-tab-close-buttons"))
++      showing_close_button_ = false;
+     available_width -= close_button_width;
```
Comment (Zoraver) [inline patches/extra/ungoogled-chromium/add-flag-to-hide-tab-close-buttons.patch:9]: `available_width` currently gets decremented here regardless of whether the close button is shown or not, which may cause a small UI regression.

**Comment by kramred** [comment]
The promotion of Chrome 97 to the stable channel for Windows, Mac and Linux [happened](https://chromereleases.googleblog.com/2022/01/stable-channel-update-for-desktop.html) (`.71` stayed)

**Review by Zoraver — APPROVED** [review]
Looks good to me. Great work Ahrotahn!

I would also like to give you guys a heads-up that I will be very busy for the next few months and will likely not have time to review the patchset for Chromium 98 in a timely manner. If you don't see input from me on pull requests over the next few months, do not be concerned.

**Review by networkException — APPROVED** [review]
Builds and runs fine for me

**Pull request merged**